renovate.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What is this file, btw? I'm usually pretty allergic to adding more top-level file junk - what's it for?

Reply:
Oh I see... we need to also add an app to our GH for this? Can we choose to put the config file in a non-in-your-face place?

Reply:
Yep, need to enable this bot in settings. We can move it into `.github`.

Reply:
d46f427fed19cd29136c4779b2c1225a132410bc